cmd/shortener/main.go

```diff
@@ -1,3 +1,106 @@
 package main
 
-func main() {}
+import (
+	"github.com/Erlast/short-url.git/internal/config"
+	"github.com/go-chi/chi/v5"
+	"io"
+	"math/rand"
+	"net/http"
+	"strings"
+)
+
+var storage map[string]string
+var conf config.Cfg
+
+func getHandler(res http.ResponseWriter, req *http.Request) {
+	id := strings.Replace(req.URL.Path, "/", "", 1)
+
+	url, err := storage[id]
```

Comment:
`err` принято использовать для ошибок, для проверок используют `ok`

```suggestion
	url, ok := storage[id]
```